### ISSUE TITLE
shim: health endpoint

### ISSUE THREAD

**Issue by pepyakin** [thrumdev/blobs#108]
Docker has a feature referred to as HEALTHCHECK. It allows running a predefined command that checks health of the container. It may be a nice-to-have for ops. It is also useful for docker-compose since it allows specifying `depends_on` for start up order and ideally the demo containers should start only after the shim is up.

**Comment by rphmeier** [comment]
this could be an RPC added to the shim?

**Comment by pepyakin** [comment]
Yes. Should be very simple. HEALTHCHECK CMD rawdogging with curl an JSONRPC endpoint works.

**Comment by gabriele-0201** [comment]
Is this achieved with #191 , as described in #128?

**Comment by pepyakin** [comment]
No, it's not. The condition for closing this issue is presence of a working integrated healthcheck.
